xml/templates/civicrm_version.tpl

```diff
@@ -1,11 +1,7 @@
 <?php
 
-function civicrmVersion( ) {ldelim}
-{include file="../../templates/CRM/common/version.tpl" assign=svnrevision}
+function civicrmVersion() {ldelim}
   return array( 'version'  => '{$db_version}',
                 'cms'      => '{$cms}',
                 'revision' => '{$svnrevision}' );
```

Comment:
'revision' has confused me since we moved to git. It seems unclear what's supposed to go in it but it's still used (possibly incorrectly) in a few places.